edb/schema/objects.py

```diff
@@ -442,9 +443,26 @@ def get_refdict_for_class(cls, refcls):
         else:
             raise KeyError(f'{cls} has no refdict for {refcls}')
 
+    @property
+    def is_schema_object(cls) -> bool:
+        return cls in ObjectMeta._schema_types
```

Comment:
Maybe instead of `is_schema_object` we can just use `issubclass` to make mypy et al happier?

Reply:
Will do in a separate refactor. This just follows what we currently have.